akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala

```diff
@@ -95,4 +96,36 @@ class DebuggingDirectivesExamplesSpec extends RoutingSpec {
       responseAs[String] shouldEqual "logged"
     }
   }
+  "logRequestResultWithResponseTime" in {
+
+    def akkaResponseTimeLoggingFunction(loggingAdapter: LoggingAdapter,
+                                        requestTimestamp: Long,
+                                        level: LogLevel = Logging.InfoLevel)(req: HttpRequest)(res: Any): Unit = {
+      val entry = res match {
+        case Complete(resp) =>
+          val responseTimestamp: Long = System.nanoTime
+          val elapsedTime: Long = (responseTimestamp - requestTimestamp) / 1000
+          val loggingString = s"""Logged Request:${req.method}:${req.uri}:${resp.status}:${elapsedTime}"""
+          LogEntry(loggingString, level)
+        case anythingElse =>
+          LogEntry(s"$anythingElse", level)
+      }
+      entry.logTo(loggingAdapter)
+    }
+    def printResponseTime(log: LoggingAdapter) = {
+      val requestTimestamp = System.nanoTime
+      akkaResponseTimeLoggingFunction(log, requestTimestamp)
+    }
+
+    val logResponseTime = DebuggingDirectives.logRequestResult(LoggingMagnet(printResponseTime(_)))
+
+    Get("/lessTime") ~> logResponseTime(complete("loggedWithGreen")) ~> check {
+      responseAs[String] shouldEqual "loggedWithGreen"
+    }
+    Get("/moreTime") ~> logResponseTime(requestContext => {
```

Comment:
Just skip this assertion, and keep the first. (maybe change it from "loggedWithGreen" to "logged" as there is no red/green anymore)